### ISSUE TITLE
Pinned headers and right margin

### ISSUE THREAD

**Issue by mundoff** [nolanlawson/SuperSaiyanScrollView#4]
The idea is to have "pinned headers" like the Contacts app in Kit Kat (Android 4.4.2):

![screen1](https://f.cloud.github.com/assets/7022847/2487637/cf340a74-b137-11e3-92c7-c7dfeb239cfa.PNG)

![screen2](https://f.cloud.github.com/assets/7022847/2487639/d4491fea-b137-11e3-9a63-1de0d05eb7f5.PNG)

and a little right margin with fast scrollbars, in Kit Kat, this scrollbar is always visible.

This webpage says how to do the right margin and provides a link where you can view the pinned headers in the source of the Contacts app:

http://www.tristanwaddington.com/2012/09/implementing-the-android-contact-applications-listview-layout/

Thank you very much.

**Comment by PeterMinin** [comment]
I don't think this library needs pinned headers. There are plenty implementations of them out there, you can find some of them here: https://play.google.com/store/apps/details?id=com.desarrollodroide.repos. I was on the contrary happy to find this light-weight library (thank you, @nolanlawson) because I don't want to include unused code into my app.

**Comment by nolanlawson** [comment]
@PeterMinin thanks for the link; that app is awesome (and contains two of my demos!). I agree that this library should stay nice and light, but if we can add this feature as an optional setting without too much extra code, I'd be all for it.
